lib/embeddings/vector_representations/all_mpnet_base_v2.rb

```diff
@@ -4,19 +4,37 @@ module DiscourseAi
   module Embeddings
     module VectorRepresentations
       class AllMpnetBaseV2 < Base
+        class << self
+          def name
+            "all-mpnet-base-v2"
+          end
+
+          def correctly_configured?
+            SiteSetting.ai_embeddings_discourse_service_api_key.present? &&
```

Comment:
API key here is optional

Reply:
There are other places where we check for this. Will remove those too